src/views/Phrases.vue

```diff
@@ -4,27 +4,26 @@ export default {
     return {
       categories: ["{Building}", "{Office Hours}", "{Professor}", "{Other}"],
       phrase: "",
-      buttonDataHasBeenAnnotated: false
-    }
+      buttonDataHasBeenAnnotated: false,
+      tokenRerenderKey: 0
+    };
   },
-  computed: { // return cached values until dependencies change
-    tokens: function () {
-      return this.tokenize(this.phrase)
+  computed: {
+    // return cached values until dependencies change
+    tokens: function() {
+      // given a string, output an array of words
+      let arr = this.phrase.split(/[ ?]/); // need to use regex to get rid of the question mark
+      return arr;
     },
     buttonData: function() {
-      return this.setupButtonData(this.tokens);
+      /// given tokens, setup button data obj for each
+      let tokens = this.tokens.filter(token => token != ""); // filters out spacess
+      return tokens.map(
+        x => (x = { token: x, category: null, isSelected: false })
+      );
```

Comment:
The return value of map automatically gets assigned. So this could be rewritten as: 
```
return tokens.map(token => { token, category: null, isSelected: false })
```